mail-integration-services/src/main/java/org/exoplatform/mailintegration/utils/RestEntityBuilder.java

```diff
@@ -53,9 +58,34 @@ public static final MailIntegrationSetting toMailIntegrationSetting(MailIntegrat
     return mailIntegrationSetting;
   }
 
-  public static final MessageRestEntity fromMessage(Message message) throws MessagingException {
+  public static final MessageRestEntity fromMessage(Message message) throws MessagingException, IOException {
     if (message != null) {
-      return new MessageRestEntity(message.getSubject(), message.getSentDate(), message.getFrom()[0].toString());
+      MessageRestEntity messageRestEntity = new MessageRestEntity();
+      messageRestEntity.setSubject(message.getSubject());
+      messageRestEntity.setSentDate(message.getSentDate());
+      messageRestEntity.setFrom(message.getFrom()[0].toString().split("<")[1].split(">")[0]);
```

Comment:
are we sure that we have always <> in the from ? otherwises we should add additional tests